src/components/Textarea.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+
+const Textbox = styled.div`
```

Comment:
This isn't really a `Textbox` more like a wrapper, it defines the positioning of its children and some styling, the actual text will be somewhere else

Reply:
sehr clever ;)